drivers/display/oled/oled.c

```diff
@@ -0,0 +1,320 @@
+	                   0x417f49091d010300,
+	                   0x1c224149493a0800,
+	                   0x417f0808087f4100,
+	                   0x0041417F41410000,
+	                   0x304041413F010100,
+	                   0x417f080c12614100,
+	                   0x417f414040406000,
+	                   0x417f420c427f4100,
+	                   0x417f420c117f0100,
+	                   0x1c22414141221c00,
+	                   0x417f490909090600,
+	                   0x0c12212161524c00,
+	                   0x417f090919694600,
+	                   0x6649494949493300,
+	                   0x0301417f41010300,
+	                   0x013f4140413f0100,
+	                   0x010f3140310f0100,
+	                   0x011f6114611f0100,
+	                   0x4141360836414100,
+	                   0x0103447844030100,
+	                   0x4361514945436100,
+	                   0x00007f4141000000,
+	                   0x0102040810204000,
+	                   0x000041417f000000,
+	                   0x0004020101020400,
+	                   0x0040404040404000,
+	                   0x0001020000000000,
+	                   0x00344a4a4a3c4000,
+	                   0x00413f4848483000,
+	                   0x003c424242240000,
+	                   0x00304848493f4000,
+	                   0x003c4a4a4a2c0000,
+	                   0x0000487e49090000,
+	                   0x00264949493f0100,
+	                   0x417f480444784000,
+	                   0x0000447d40000000,
+	                   0x000040443d000000,
+	                   0x417f101824424200,
+	                   0x0040417f40400000,
+	                   0x427e027c027e4000,
+	                   0x427e4402427c4000,
+	                   0x003c4242423c0000,
+	                   0x00417f4909090600,
+	                   0x00060909497f4100,
+	                   0x00427e4402020400,
+	                   0x00644a4a4a360000,
+	                   0x00043f4444200000,
+	                   0x00023e4040227e40,
+	                   0x020e3240320e0200,
+	                   0x021e6218621e0200,
+	                   0x4262140814624200,
+	                   0x0143453805030100,
+	                   0x004662524a466200,
+	                   0x0000083641000000,
+	                   0x0000007f00000000,
+	                   0x0000004136080000,
+	                   0x0018080810101800,
+	                   0xAA55AA55AA55AA55
+};
+
+/***************************************************************************//**
+ * @brief Turns display on/off.
+ *
+ * @param device - The device structure.
+ *        on_off - DISP_ON
+ *                 DISP_OFF
+ *
+ * @return SUCCESS in case of success, negative error code otherwise.
+*******************************************************************************/
+int32_t display_on_off(struct ssd_1306_dev *device, uint8_t on_off)
+{
+	int32_t ret = SUCCESS;
+	uint8_t command;
+
+	ret |= gpio_set_value(device->dc_pin, DC_CMD);
+
+	command = (on_off == DISP_ON) ? 0xAF : 0xAE;
```

Comment:
what does `0xAF`, `0xAE` mean? maybe some suggestive macros should be added. This applies to most of the raw values used inside the driver.